app/src/shared/components/Toolbar/Toolbar.stories.jsx

```diff
@@ -34,24 +35,41 @@ const toolbarActions = {
     },
 }
 
-story('Toolbar').addWithJSX('basic', () => (
-    <Fragment>
-        <Toolbar
-            actions={toolbarActions}
-            altMobileLayout
-        />
-        <Content>Lorem ipsum dolor sit emat.</Content>
-    </Fragment>
-))
+story('Toolbar')
+    .addParameters({
+        chromatic: {
+            // Take 5 snapshots
+            viewports: [SM - 1, SM, MD, LG, XL],
+        },
+    })
+    .addWithJSX('basic', () => (
+        <Fragment>
+            <Toolbar
+                actions={toolbarActions}
+                altMobileLayout
+            />
+            <Content>Lorem ipsum dolor sit emat.</Content>
+        </Fragment>
+    ))
+
+const Autoscrolled = ({ children }: any) => {
+    const ref = useRef()
 
-const Autoscrolled = (props: any) => {
     useEffect(() => {
-        if (typeof window !== 'undefined') {
```

Comment:
Wanted to present a natural post-scroll behaviour and it didn't work. Trying a different approach. If this doesn't work I'mma remove the story.